### ISSUE TITLE
React Native PayStack for Expo

### ISSUE THREAD

**Issue by davidteejay** [tolu360/react-native-paystack#44]
Hi. Looking through the ReadMe, it seems the package requires linking, which cannot work with expo. Is there a way to make this work with expo?

**Comment by larrybek** [comment]
@davidteejay good point I hope someone can take care of its. Have you found a way to integrate PayStack in expo?

**Comment by davidteejay** [comment]
No, I have not. I've always had to eject to ExpoKit or switch to the bare workflow

**Comment by larrybek** [comment]
@davidteejay ExpoKit is already depreciated, yet bare workflow is the way of yours, will keep posted if I find an alternative way.

**Comment by davidteejay** [comment]
Nice! Thank you

**Comment by ihemegbulam** [comment]
Why is there no solution for expo on this subject?

**Comment by elouisramsey** [comment]
Does this now work for expo?

**Comment by davidteejay** [comment]
Expo now has the bare workflow, so this package should work with the expo bare workflow.

**Comment by onyijne** [comment]
With EAS and expo development client it's now possible to use with expo.
After installation, create a development build with eas, then use the build for development as you would with any other react native package.

**Comment by BrightonNgema** [comment]
Can someone with a solution please show us how they implemented it.

**Comment by Temi-Dire** [comment]
Can someone with a solution please show us how they implemented it.

**Comment by Yembot31013** [comment]
I will see what I can do and then provide a solution for those that still want to use **expo managed flow**, I guess the solution will be to create pre-build plugin that will automate the linking.
